internal/gengapic/custom_operation.go

```diff
@@ -130,6 +130,28 @@ func (g *generator) customOperationType() error {
 	p("}")
 	p("")
 
+	// Wait
+	p("// Wait blocks until the operation is complete, polling regularly")
+	p("// after an intial period of backing off between attempts.")
+	p("func (o *%s) Wait(ctx context.Context, opts ...gax.CallOption) error {", opName)
+	p("	 bo := gax.Backoff{")
+	p("    Initial: 1 * time.Second,")
+	p("    Max:     time.Minute,")
```

Comment:
nit: why not `1 * time.Minute` here like in the line above for `Second`?

Reply:
Good catch. Done.